tracpro/static/less/trac.less

```diff
@@ -203,4 +203,44 @@ footer {
 /* Override datepicker colors */
 .ui-datepicker select.ui-datepicker-month, .ui-datepicker select.ui-datepicker-year {
   color: #000000;
-}
\ No newline at end of file
+}
```

Comment:
I noticed some differences from the mockup:

* On the mockup, the circles had a very light circle drawn around them just a few pixels outside the colored circle.
* The text in the unselected circles was dark rather than light.
* The labels were bold.
* The circles appear smaller in the mockup, though I haven't measured them.